internal/pkg/manifest/backend_svc.go

```diff
@@ -126,9 +126,20 @@ func (s *BackendService) BuildArgs(contextDir string) map[string]*DockerBuildArg
 	return buildArgs
 }
 
-// EnvFile returns the location of the env file against the ws root directory.
-func (s *BackendService) EnvFile() string {
-	return aws.StringValue(s.TaskConfig.EnvFile)
+// EnvFiles returns the locations of all env files against the ws root directory.
+// This method returns a map[string]string where the keys are container names
+// and the values are either env file paths or empty strings.
+func (s *BackendService) EnvFiles() map[string]string {
```

Comment:
Looks like the logic for `EnvFiles` funcs are almost identical. Can we possibly reduce the duplication?